### DISCUSSION TITLE
[BUG] No alarm sound - solved

### DISCUSSION THREAD

**Discussion by miguelvalente** [JaDogg/pydoro#266] [General]
**Describe the bug**
No alarm sound after timer ends on either break or work.

**To Reproduce**
run pydoro on a new terminal
start running timer
No alarm after timer ends

**Expected behavior**
Alarm sounds after timer ends during break or work

**Desktop :**
 - OS: Arch Linux
 - DM: dwm

**Comment by JaDogg** [comment]
Try installing `pygame` and it will use pygame if found. Also how did you install it? what command did you use?

**Comment by JaDogg** [comment]
Also see this issue for how someone else got it working in Arch -- https://github.com/JaDogg/pydoro/issues/53

**Comment by JaDogg** [comment]
@miguelvalente did that solve your problem?

**Comment by miguelvalente** [comment]
Tried:

- Can't install PyGObject following https://pygobject.readthedocs.io/en/latest/getting_started.html#arch-logo-arch-linux
- pygame was already installed
- Installing the package mentioned in the issue

Problem still persists.

**Reply by JaDogg** [reply]
@miguelvalente can you let me know if the issue is sorted or not with the new update? 😄

**Comment by JaDogg** [comment]
Try uninstalling and installing from setup.py into same environment (by cloning the repo) that you have pygame installed.  Also ensure you are running Python3.6+ and not Python2. (this installs development version)

I do not have Arch installed to try it myself. 

Also try `--audio-check` command after installing from setup.py and see if it works.

**Comment by JaDogg** [comment]
Can't install PyGObject following https://pygobject.readthedocs.io/en/latest/getting_started.html#arch-logo-arch-linux -- why not? can you share error messages? It's not possible to troubleshoot things without full error messages. (Please copy paste text, no screenshots). It might be better to reach out to pygobject forums on this or ask in an Arch Linux forum.

**Comment by JaDogg** [comment]
Alternatively I can do a new release (Probably next week) and you can install that.. one in PyPI is 2 years old and doesn't have the pygame fallback. Installing from setup.py should work for you meanwhile. @miguelvalente

**Comment by JaDogg** [comment]
@miguelvalente I did a new release try upgrading to that.. it should have the pygame fallback. and ways to debug audio issues :)

**Comment by miguelvalente** [comment]
Hi @JaDogg ,

I re-installed `pygame` and `pydoro`. Sound works now. 

Was busy last couple days so it took me a while to get back to you. Also sorry for the lack of info on the `PyGObject` errors,  its really frustrating to spend upwards of an hour fixing an issue on a productivity app, made me question my sanity.

Thanks, smooth  as butter now :).
